polybar-scripts/anyconnect-status/README.md

```diff
@@ -0,0 +1,37 @@
+# Script: anyconnect-status
+
+Checks if an interface named `cscotun` exists, if not display a red
+`VPN DISCONNECTED` message. If connected, displays it's IP address.
+
+![connected](screenshots/connected.png)
+![disconnected](screenshots/disconnected.png)
+
+## Color and glyph
+
+The contribution guidelines require colors to be stripped. Like the screen
+shots? Here's the exact script:
+
+```bash
+#!/usr/bin/env bash
+
+IFACE=$(ifconfig | grep cscotun | awk '{print $1}')
+
+if [[ $IFACE == "cscotun"* ]];
+then
+    echo "%{u#55aa55}$(ifconfig cscotun0 | grep inet | awk '{print $2}' | cut -f2 -d':')%{u-}"
```

Comment:
Add this line as a comment in your script, so that everyone can simply uncomment this part if he wants colors. Otherwise we have a second version of the same script in our `README.md`. Maybe not the best idea.